### DISCUSSION TITLE
Elastic net results from lifeline and scikit-survival are different

### DISCUSSION THREAD

**Discussion by AnonTendim** [sebp/scikit-survival#381] [Q&A]
Trying to compare the elastic net results from lifelines and sckie-learn survival by running the following code.
```
from sksurv.datasets import load_breast_cancer
X, y = load_breast_cancer()
Xt = OneHotEncoder().fit_transform(X)
Xt.round(2).head()
coxnet_pred = CoxnetSurvivalAnalysis(alphas = [0.01], l1_ratio=1, fit_baseline_model=True)
coxnet_pred.fit(Xt, y)
print(coxnet_pred.coef_)

df = Xt.copy()
df['death'] = [yy[0] for yy in y]
df['time_diag_end'] = [yy[1] for yy in y]

cph = CoxPHFitter(penalizer=0.01, l1_ratio=1.0) # sparse solutions,
cph.fit(df, 'time_diag_end', 'death')
cph.print_summary()
```
The parameters are the same but the coefficients of the output model do not match. Is it due to difference in implementation or I used one of the models incorrectly? Thanks!

**Comment by sebp** [comment]
I'm not familiar with with lifelines. Implementation could be one explanation, and also that the penalization term might not be identical: it could be scaled differently such that 0.01 means different things.
